src/sortStudents.ts

```diff
@@ -1,16 +1,71 @@
 
 export interface Student {
   // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
   // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
-
-export function sortStudents(students, sortBy, order) {
+export function sortStudents(
+  students: Student[], sortBy: SortType, order: SortOrder,
+): object[] {
   // write your function
+  const sortedStudents: Student[] = [...students];
+
+  sortedStudents.sort((student1, student2) => {
```

Comment:
```suggestion
  sortedStudents.sort((firstStudent, secondStudent) => {
```
this naming is better and more descriptive :)